devtools.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'rspec',        '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-core',   '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-its',    '~> 1.2.0'
-  gem.add_runtime_dependency 'rake',         '~> 10.4.2'
+  gem.add_runtime_dependency 'rake',         '~> 11.1.x'
```

Comment:
My impression was that the `.x` signaled that you shouldn't allow updating to a minor version (in this case `11.2`) but you are fine with any patch level version. If I make it `11.1.2` then we are just going to let that go stale even if `11.1.3` is released so I think it is fine to just say "any minor version"